pages/Product/index.jsx

```diff
@@ -18,21 +18,13 @@ import connect from './connector';
  */
 class Product extends Component {
   static propTypes = {
-    resetCurrentProduct: PropTypes.func.isRequired,
     name: PropTypes.string,
   };
 
   static defaultProps = {
     name: null,
   };
 
-  /**
-   * Component will unmount and reset the current product.
-   */
-  componentWillUnmount() {
```

Comment:
Because you removed the lifecycle hook, the component is a big linter error now  - `read/prefer-stateless-function`